osprey_worker/src/osprey/worker/lib/storage/stored_execution_result.py

```diff
@@ -482,9 +483,17 @@ def get_many(
 
 def bootstrap_execution_result_storage_service() -> ExecutionResultStorageService:
     """Create an ExecutionResultStorageService with the configured storage backend."""
-    from osprey.worker.adaptor.plugin_manager import bootstrap_execution_result_store
+    from osprey.worker._stdlibplugin.execution_result_store_chooser import get_rules_execution_result_storage_backend
     from osprey.worker.lib.singletons import CONFIG
 
     config = CONFIG.instance()
-    storage_backend = bootstrap_execution_result_store(config)
+
+    storage_backend_type = ExecutionResultStorageBackendType(
+        config.get_str('OSPREY_EXECUTION_RESULT_STORAGE_BACKEND', 'none')
```

Comment:
```suggestion
        config.get_str('OSPREY_EXECUTION_RESULT_STORAGE_BACKEND', 'none').upper()
```
do python enums act case insensitive ? if not, all the types seem to only accept upper so we might as well `upper()` here for robustness. if so, ignore me 😆

Reply:
so ig `auto()` [makes the strings lower](https://docs.python.org/3/library/enum.html#enum.StrEnum) so i think we atually want `.lower()`

Reply:
```suggestion
        config.get_str('OSPREY_EXECUTION_RESULT_STORAGE_BACKEND', 'none').lower()
```